<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SG</class>
 <widget class="QMainWindow" name="SG">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sound Graph</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #22222E</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>55</y>
      <width>1024</width>
      <height>1</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #FFD700;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="icon">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFD700</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>2</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Slab SemiBold</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFD700; background-color: #2D2D3D</string>
    </property>
    <property name="text">
     <string>Sound Graph</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>336</width>
      <height>576</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2D2D3D</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="input_type">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>315</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Slab</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #2D2D3D;
border: 1px solid #FFD700;
color: white;
</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string>Файл</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Файл</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Звуки устройства</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="settings_text">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>58</y>
       <width>330</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Slab</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #FFD700; background-color: #2D2D3D</string>
     </property>
     <property name="text">
      <string>Настройки программы</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="fopt_text">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>95</y>
       <width>330</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Slab</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #FFD700; background-color: #2D2D3D</string>
     </property>
     <property name="text">
      <string>1. Тип ввода анализируемого звука</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="sopt_text">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>195</y>
       <width>330</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Slab</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #FFD700; background-color: #2D2D3D</string>
     </property>
     <property name="text">
      <string>2. Настроки типа вывода</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="path">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>230</y>
       <width>125</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Slab</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #2D2D3D;
	border: 1px solid #FFD700;
	color: white;
}

QPushButton:hover {
	background-color: #B59900;
}

 QPushButton:!hover {
	background-color: #2D2D3D;
}

QPushButton:pressed {
	background-color: #FFD700;
}
</string>
     </property>
     <property name="text">
      <string>Выбрать файл</string>
     </property>
    </widget>
    <widget class="QLabel" name="path_text">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>265</y>
       <width>45</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Slab</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #FFD700; background-color: #2D2D3D</string>
     </property>
     <property name="text">
      <string>Путь:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="path_ch_text">
     <property name="geometry">
      <rect>
       <x>62</x>
       <y>266</y>
       <width>261</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #FFD700; background-color: #2D2D3D;  text-decoration: underline;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="mic_ins">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>220</y>
       <width>330</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Slab</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #FFD700; background-color: #2D2D3D</string>
     </property>
     <property name="text">
      <string>Выберите устройство записи:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="mic_type">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>245</y>
       <width>305</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Slab</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #2D2D3D;
border: 1px solid #FFD700;
color: white;
</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="start_sound">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>300</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Slab</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #2D2D3D;
	border: 1px solid #FFD700;
	color: white;
}

QPushButton:hover {
	background-color: #B59900;
}

 QPushButton:!hover {
	background-color: #2D2D3D;
}

QPushButton:pressed {
	background-color: #FFD700;
}
</string>
     </property>
     <property name="text">
      <string>Старт</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_sound">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>300</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Slab</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #2D2D3D;
	border: 1px solid #FFD700;
	color: white;
}

QPushButton:hover {
	background-color: #B59900;
}

 QPushButton:!hover {
	background-color: #2D2D3D;
}

QPushButton:pressed {
	background-color: #FFD700;
}
</string>
     </property>
     <property name="text">
      <string>Стоп</string>
     </property>
    </widget>
    <widget class="QPushButton" name="start_graphing">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>316</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Slab</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #2D2D3D;
	border: 1px solid #FFD700;
	color: white;
}

QPushButton:hover {
	background-color: #B59900;
}

 QPushButton:!hover {
	background-color: #2D2D3D;
}

QPushButton:pressed {
	background-color: #FFD700;
}
</string>
     </property>
     <property name="text">
      <string>Старт анализа звука</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>195</y>
      <width>315</width>
      <height>1</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #FFD700;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>195</y>
      <width>1</width>
      <height>100</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #FFD700;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>195</y>
      <width>1</width>
      <height>100</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #FFD700;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>295</y>
      <width>316</width>
      <height>1</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #FFD700;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>295</y>
      <width>1</width>
      <height>40</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #FFD700;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>295</y>
      <width>1</width>
      <height>40</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #FFD700;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>335</y>
      <width>316</width>
      <height>1</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #FFD700;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>0</y>
      <width>1</width>
      <height>577</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #FFD700;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>line</zorder>
   <zorder>icon</zorder>
   <zorder>name</zorder>
   <zorder>line_2</zorder>
   <zorder>line_3</zorder>
   <zorder>line_4</zorder>
   <zorder>line_5</zorder>
   <zorder>line_6</zorder>
   <zorder>line_7</zorder>
   <zorder>line_8</zorder>
   <zorder>line_9</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
